bigcommerce/resources/webhooks.py

```diff
@@ -0,0 +1,6 @@
+from .base import *
+
+
```

Comment:
I could but, I think it would be a style violation. When I remove it and run `pep8` against the file:

    ➤ pep8 bigcommerce/resources/webhooks.py
    bigcommerce/resources/webhooks.py:3:1: E302 expected 2 blank lines, found 1